tests/utils_tests/test_datastructures.py

```diff
@@ -55,6 +55,10 @@ def test_len(self):
         s.add(2)
         self.assertEqual(len(s), 2)
 
+    def test_reversed(self):
```

Comment:
I would put this test method right before `test_contains()` to match the order of the methods in the class.

Reply:
Nice! 441bfded3f1028a04ba1c547f5bf588302e4e407